docs/layouts/shortcodes/generated/mongodb_sync_database.html

```diff
@@ -62,7 +62,7 @@
     </tr>
     <tr>
         <td><h5>--mongodb_conf</h5></td>
-        <td>The configuration for Flink CDC MongoDB sources. Each configuration should be specified in the format "key=value". hosts, username, password, database are required configurations, others are optional. See its <a href="https://nightlies.apache.org/flink/flink-cdc-docs-master/docs/connectors/flink-sources/mongodb-cdc/#connector-options">document</a> for a complete list of configurations.</td>
```

Comment:
`https://nightlies.apache.org/flink/flink-cdc-docs-master/docs/connectors/flink-sources/mongodb-cdc/#connector-options`  The document address is correct, why change that?